src/site/markdown/server/manage-webadmin.md

```diff
@@ -341,6 +342,24 @@ Response codes:
 
  - 200: The user name list was successfully retrieved
 
+### Retrieving the list of allowed `From` headers for a given user
+
+```
+curl -XGET http://ip:port/users/givenUser/allowedFromHeaders
+```
+
+The answer looks like:
+
+```
+["user@domain.tld","alias@domain.tld"]
```

Comment:
Hello,
Do those emails contain only user aliases or both user and domain aliases?